src/Symfony/Component/HttpKernel/HttpCache/Store.php

```diff
@@ -387,16 +387,22 @@ private function save($key, $data)
 
             $tmpFile = tempnam(dirname($path), basename($path));
             if (false === $fp = @fopen($tmpFile, 'wb')) {
+                @unlink($tmpFile);
```

Comment:
you cannot unlink an opened file on Windows

Reply:
I think the file is not open when fopen returns false.

Reply:
That's quite possible :)